src/components/task/MemberTaskAdminModal.tsx

```diff
@@ -185,10 +185,24 @@ const MemberTaskAdminModal: React.FC<
         if (!formMemberId) return handleError({ message: 'value memberId is necessary' })
         const memberTaskId = memberTask?.id
         const memberTaskMeetId = memberTask?.meet?.id || null
+        const memberTaskMeetingGateway = memberTask?.meetingGateway
+        const memberTaskExecutorId = memberTask?.executor?.id
+        const memberTaskMemberId = memberTask?.member?.id
+        const memberTaskMeetingHours = memberTask?.meetingHours
+        const memberTaskDueAtTime = toMillisecond(memberTask?.dueAt)
+        const formDueAtTime = toMillisecond(formDueAt?.toDate())
+
         let toBeUsedServiceId: string | null = null
         let updatedMeetId = memberTask?.meet?.id || null
 
-        if (hasMeeting) {
+        if (
+          hasMeeting &&
+          (memberTaskDueAtTime !== formDueAtTime ||
+            memberTaskExecutorId !== formExecutorId ||
+            memberTaskMemberId !== formMemberId ||
+            memberTaskMeetingGateway !== formMeetingGateway ||
+            memberTaskMeetingHours !== formMeetingHours)
```

Comment:
these condition are unnecessary.

There are two situations where a meeting will be deleted:

1. When there is no meeting.
2. When there is a meeting and meeting-related information has been modified.

Reply:
Thanks to review, it has been adjusted and refactored.